### DISCUSSION TITLE
[WARN]  http: TLS handshake error from x.x.x.x:pppp acme/autocert: host "z.z.z.z" not configured in HostWhitelist

### DISCUSSION THREAD

**Discussion by OnkelTem** [umputun/remark42#1717] [Q&A]
I see a bunch of warnings like this, how to fix it? I'm not sure how to configure that `HostWhilelist`.

**Comment by paskal** [comment]
I presume the error is the same as one described in #1713, feel free to reopen if I'm wrong.

**Reply by OnkelTem** [reply]
It's not really an error, it's a warning and it doesn't seem to block comments from working. But I just wonder why I see this. Where is HostWhitelist?

**Reply by paskal** [reply]
It's taken from [REMARK_URL](https://remark42.com/docs/configuration/parameters/) and used only if you use `SSL_TYPE=auto` to generate the HTTPS certificate using Let's Encrypt at the start of the application and then once in a while.

Code where HostWhitelist is mentioned:

<img width="653" alt="image" src="https://github.com/umputun/remark42/assets/712534/cb1d8105-2537-4a2a-bb3a-59881a5d15b1">
